src/PaymentGateways/PayPalCommerce/AjaxRequestHandler.php

```diff
@@ -178,6 +178,7 @@ public function createOrder() {
 
 		$data = [
 			'formId'              => $formId,
+			'formTitle'           => give_payment_gateway_item_title( [ 'post_data' => $postData ] ),
```

Comment:
Same here, there's likely a character limit and long form titles may cause this to break.

Reply:
fixed: https://github.com/impress-org/givewp/compare/f56cb66a7ea2...4d372e91cff2